__tests__/endpoint.test.js

```diff
@@ -108,25 +108,67 @@ describe('Endpoint Tests', () => {
                 })
             })
             test('GET:200', () => {
-                return request(app).get('/api/artciles/2/comments').expect(200)
+                return request(app).get('/api/articles/2/comments').expect(200)
                 .then(({body}) => {
                     expect(body.data).toEqual([])
                 });
             });
             test('GET:400 sends an appropriate status and error message when given an invalid path', () => {
-                return request(app).get('/api/artciles/not-a-number/comments').expect(400)
+                return request(app).get('/api/articles/not-a-number/comments').expect(400)
                 .then((response) => {
                     expect(response.body.msg).toBe('Bad Request')
                 });
             });
             test('GET:404 responds with an appropriate status and error message when given a non-existent id', () => {
                 return request(app)
-                  .get('/api/artciles/999999/comments')
+                  .get('/api/articles/999999/comments')
                   .expect(404)
                   .then((response) => {
                     expect(response.body.msg).toBe('article ID does not exist');
                   });
               });
         })
+        
     })
+    describe('POST /api/articles/:article_id/comments', () => {
+        test('POST:201 takes an article id and inserts a new comment to the db and responds with the posted comment', () => {
+            const newComment = {
+              username: 'butter_bridge',
+              body: 'This is a test'
+            };
+            return request(app)
+              .post('/api/articles/4/comments')
+              .send(newComment)
+              .expect(201)
+              .then((response) => {
+                expect(response.body.comment.article_id).toBe(4);
+                expect(response.body.comment.author).toBe('butter_bridge');
+                expect(response.body.comment.body).toBe('This is a test');
+              });
+          });
```

Comment:
Nice start to this first test, but we should be expecting every property in the comment not just the ones we know the values of. This is to ensure we get back everything correctly.